docs/local-development.md

```diff
@@ -604,7 +604,7 @@ We use [eslint](http://eslint.org/) to maintain our JavaScript styles. The [.esl
 To automatically fix many errors run the command:
 
 ```bash
-yarn run lint-fix
```

Comment:
This command didn't make a ton of sense anymore in the docs or package.json as the normal JS lint ran the `--fix` option.